pkg/handlers/platform_license.go

```diff
@@ -0,0 +1,53 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"os"
+
+	"github.com/replicatedhq/kotsadm/pkg/license"
+	"github.com/replicatedhq/kotsadm/pkg/logger"
+)
+
+const (
+	DefaultAPIEndpoint = "https://replicated.app"
```

Comment:
this kinda sucks cause rli files do not contain the api endpoint like license yaml does